demo/chbench/dc.sh

```diff
@@ -281,6 +282,10 @@ nuke_docker() {
 
 # Long-running load test
 load_test() {
+    dc_stop chbench
+    dc_stop materialized peeker
+    runv docker exec -it chbench_kafka_1 kafka-topics --delete --bootstrap-server localhost:9092 --topic mysql.tpcch.*
```

Comment:
could you move this into a `drop_kafka_topics` function, and put the topic in quotes: `--topic "mysql.tpcch.*"`, with default bash configs that `*` will expand if someone happens to run this in a directory that has files that start with `mysql.tpcch.`